modules/core/server/views/layout.server.view.html

```diff
@@ -16,7 +16,8 @@
 
   <!-- Application CSS Files -->
   {{#each cssFiles}}<link rel="stylesheet" href="/{{this}}">{{/each}}
-</head>
```

Comment:
you accidentally deleted the head closing tag

Reply:
That isn't showing on my local, it just got moved down a line with the bootstrap href tag.